distrobox-enter

```diff
@@ -218,9 +218,10 @@ generate_command() {
 	# Start container from working directory. Else default to home. Else do /.
 	# pass distrobox-enter path, it will be used in the distrobox-export tool.
 	workdir="$(echo "${PWD:-${HOME:-"/"}}" | sed -e 's/"/\\\"/g')"
+	container_available_workdir="/run/host/${workdir}"
```

Comment:
let's reuse `workdir` variable, no need for a new one

Reply:
Done.